builtin/game/features.lua

```diff
@@ -13,6 +13,7 @@ core.features = {
 	object_use_texture_alpha = true,
 	object_independent_selectionbox = true,
 	httpfetch_binary_data = true,
+	set_fov = true,
```

Comment:
this can be detected using `if player.set_fov then` (assuming you have a playerref at hand)
so this doesn't need to be here

Reply:
True, removed.